pkg/engine/apiPath.go

```diff
@@ -113,7 +113,7 @@ func NewAPIPath(path string) (*APIPath, error) {
 		return &APIPath{
 			Root:         paths[0],
 			Group:        paths[1],
-			Version:      paths[2],
+			Version:      paths[1] + "/" + paths[2],
```

Comment:
Instead of appending the `Group` here, can we do that while constructing the path in the `String()` method?  That seems a bit cleaner to me, as all path assembly would be handled in that method. Or, are these some reasons why it cannot be done there?

Reply:
In places like [here](https://github.com/kyverno/kyverno/blob/db49046e26b676b724c62c01d7a7bc58b5c38630/pkg/engine/jsonContext.go#L134) the Version attribute is what is passed down and used to find resources.

Reply:
Got it, thanks!